src/push_api_clientpy/platformclient.py

```diff
@@ -103,11 +107,26 @@ class BatchUpdateDocuments:
     addOrUpdate: list[Document]
     delete: list[BatchDelete]
 
+@dataclass
+class BackoffOptions:
+    retry_after: int = DEFAULT_RETRY_AFTER
+    max_retries: int = DEFAULT_MAX_RETRIES
+    time_multiple: int = DEFAULT_TIME_MULTIPLE
+
 
 class PlatformClient:
-    def __init__(self, apikey: str, organizationid: str):
+    def __init__(self, apikey: str, organizationid: str, backoff_options: BackoffOptions, session = requests.Session()):
         self.apikey = apikey
         self.organizationid = organizationid
+        self.backoff_options = backoff_options
+
+        self.retries = Retry(status=self.backoff_options.max_retries,
+                        status_forcelist=[429],
+                        backoff_factor=self.backoff_options.time_multiple,
+                        backoff_jitter=self.backoff_options.retry_after
```

Comment:
It turns out that this documented field doesn't actually exist on the latest version of urllib3 🤦 Investigating work-arounds now...

Reply:
EDIT: That didn't seem to resolve the problem. Seeing as this is a known limitation within a very widely used library, it's not deal-breaking. I'll remove time_multiple support and move on.